components/property_access.rst

```diff
@@ -257,18 +257,24 @@ can use setters, the magic ``__set()`` method or properties to set values::
             $this->$property = $value;
         }
 
-        // ...
+        public function getLastName() {
+            return $this->lastName;
+        }
+
+        public function getChildren() {
+            return $this->children;
+        }
```

Comment:
I would move both methods between `setLastName()` and `__set()`. And to match the code style of the Symfony code the opening curly brace should be on its own line. However, we can easily make this change while merging your PR.